vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -282,6 +284,9 @@ class LEDGER_EXPORT Ledger {
       const uint64_t to_timestamp_seconds,
       ledger::ConfirmationsHistoryCallback callback) = 0;
   virtual void GetRewardsInternalsInfo(ledger::RewardsInternalsInfo* info) = 0;
+  virtual void RefreshPublisher(
+      ledger::OnRefreshPublisherCallback callback,
```

Comment:
nit: can we do first publisher_key and then callback

Reply:
Done